src/Cake.Common/Tools/OctopusDeploy/PushPackageSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
```

Comment:
@agc93 none of these using statements are actually required, and can be removed.